core/src/main/java/org/apache/calcite/util/graph/Graphs.java

```diff
@@ -134,7 +135,44 @@ public int size() {
       if (from.equals(to)) {
```

Comment:
I think we can just remove this method. Nobody is using it anymore.

Reply:
Removed. Maybe we can add it back some time in the future, if we find a requirement for it.